content/en/docs/reference/glossary/duration.md

```diff
@@ -4,11 +4,18 @@ id: duration
 date: 2024-10-05
 full_link:
 short_description: >
-  A time interval specified as a string in the format accepted by Go's [time.Duration](https://pkg.go.dev/time), allowing for flexible time specifications using various units like seconds, minutes, and hours.
-aka:
+  A string value representing an amount of time.
 tags:
 - fundamental
 ---
-In Kubernetes APIs, a duration must be non-negative and is typically expressed with a suffix. 
-For example, `5s` for five seconds or `1m30s` for one minute and thirty seconds.
+A string value representing an amount of time.
 
+<!--more-->
+
+The format of a duration is based on the [`time.Duration`](https://pkg.go.dev/time#Duration)
+type from the Go programming language.
+
+For example: `5s` for five seconds or `1m30s` for one minute and thirty seconds.
+
+In Kubernetes APIs, a duration must be non-negative and is typically expressed with a suffix.
+The valid time units are "ns", "µs" (or "us"), "ms", "s", "m", and "h".
```

Comment:
This explanation is simplified but may be misleading. For example, `1m30s` is a valid duration, right? So it is not "a suffix".

Reply:
The original description of "duration", from the source code, is this:

> A duration string is a possibly signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".

See the difference?

Reply:
How's the new version?

Reply:
It is good.